locations/spiders/kfc_ru.py

```diff
@@ -0,0 +1,54 @@
+import scrapy
+from scrapy.http import JsonRequest
+
+from locations.categories import apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_EN, OpeningHours
+
+
+class KfcRUSpider(scrapy.Spider):
+    name = "kfc_ru"
+    allowed_domains = ["kfc.digital"]
+    item_attributes = {"brand": "KFC", "brand_wikidata": "Q524757"}
+
+    def start_requests(self):
+        yield JsonRequest("https://api.kfc.digital/api/store/v2/store.get_restaurants?showClosed=false")
+
+    def parse(self, response):
+        for poi in response.json().get("searchResults"):
+            store = poi.get("storePublic")
+            if store.get("brand") != "kfc":
+                # TODO: add Rostics (https://xn--h1aekhgce.xn--p1ai/) to NSI/Wikidata,
+                #       it will slowly replace KFC in RU
+                continue
+            item = DictParser.parse(store)
+            item["ref"] = store.get("storeId")
+            item["name"] = store.get("title", {}).get("ru")
+            item["extras"]["name:en"] = store.get("title", {}).get("en")
+            # There are a few test stores in the data, skip them
+            if item["name"].lower().startswith("тест"):
+                continue
+            contacts = store.get("contacts")
+            item["phone"] = contacts.get("phoneNumber")
+            item["addr_full"] = contacts.get("streetAddress", {}).get("ru")
+            item["city"] = contacts.get("city", {}).get("ru")
+            lat, lon = contacts.get("coordinates", {}).get("geometry", {}).get("coordinates", [])
+            item["lat"] = lat
+            item["lon"] = lon
```

Comment:
It almost got me!

Usually GeoJSON is lon lat. But it's lat lon here.

https://data.alltheplaces.xyz/map.html?show=https://data.alltheplaces.xyz/ci/5541951718/kfc_ru/output.geojson